itenium-socks/src/main.ts

```diff
@@ -1,6 +1,9 @@
+/// <reference types="@angular/localize" />
```

Comment:
Dit is voor vertalingen, niet zozeer voor formatting van numbers
Zie ook de oefn `Internationalization` waar dit terug komt